### PR TITLE
Move out hardwareInfo card, more error handling

### LINKED ISSUE #264: Add hardware_info Event to Backend-mock

### PR DESCRIPTION
**Pull request by cstenglein**
Merge after #267 

Closes #264

### CODE REVIEW

File: src/components/Home/HardwareCard/HardwareCard.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  }
+
+  const {
+    cpu_overall_percent,
+    temperatures_celsius,
+    disks,
+    vram_usage_percent,
+  } = hardwareInfo;
+
+  const system_temp = temperatures_celsius.system_temp;
```
Comment (escapedcat) [inline src/components/Home/HardwareCard/HardwareCard.tsx:30]: Do we internally want to stick to lowerCamelCase?

File: src/components/Home/HardwareCard/HardwareCard.tsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+  if (!isNaN(+hddUsedGB) && !isNaN(+hddFreeGB)) {
+    hddUsedPercent = ((+hddUsedGB / +hddFreeGB) * 100).toFixed(2);
+  }
+
+  return (
+    <div className="bd-card mt-10 w-full transition-colors lg:mt-0 lg:ml-2 lg:w-1/2">
+      <div className="flex items-center text-lg font-bold">
+        {t("hardware.header")}
+      </div>
+      <article className="flex flex-row overflow-hidden py-4">
```
Comment (escapedcat) [inline src/components/Home/HardwareCard/HardwareCard.tsx:56]: nit: Don't know how this screen looks like but acrticle/h6/div might be a bit too much. Mayeb it's a `ul` or a `dl`? But non-blocking.

File: src/components/Home/HardwareCard/HardwareCard.tsx (hunk 1 above)
Comment (cstenglein) [reply src/components/Home/HardwareCard/HardwareCard.tsx:30]: My bad, sorry :+1: 
Thank you!

File: src/components/Home/HardwareCard/HardwareCard.tsx (hunk 2 above)
Comment (cstenglein) [reply src/components/Home/HardwareCard/HardwareCard.tsx:56]: Will look into it

**Comment by cstenglein** [comment]
image

HardwareCard on the right. The stuff with the `article` and `h6` is the same throughout the cards, so I'll leave it currently.

Maybe I'll need to do some accessibility work later on :P

**Pull request merged**